### ISSUE TITLE
@reducedDims not found in scater>1.14.5

### ISSUE THREAD

**Issue by ansonrel** [plger/pipeComp#6]
Hi PL, 

I tried to run the `clust.scran` in the `inst/extdata` folder of the package and I got this: 

```
> clust_id <- lapply(sces, function(x) {
+   clust.scran(x, method = "fast_greedy")
+ })
Error in clust.scran(x, method = "fast_greedy") : 
  no slot of name "reducedDims" for this object of class "SingleCellExperiment"
```

In the function, the reduced dimensions are called as 

```
ds@reducedDims
```

And this slot doesn't exist. Is it possible that `scater` removed this slot ?? (I'm working with the latest version, 1.14.5). 

In any case, using `ds@int_colData@listData$reducedDims` works. 

Maybe worth specifying the required version of `scater` or updating the function call ? 

Thanks!

**Comment by plger** [comment]
Hi,
thanks for letting me know...
that should anyway be replaced by the proper method, i.e. `reducedDim(ds)` (then specifying the type)
Pierre-Luc

**Comment by ansonrel** [comment]
Corrected in #7 .
